src/simplestreams/simple_streams_manifest.cpp

```diff
@@ -61,18 +61,11 @@ QString latest_version_in(const QJsonObject& versions)
     return max_version;
 }
 
-QString derive_unpacked_file_path_prefix_from(const QString& image_location)
+QString derive_unpacked_file_path_prefix_from(const QString& image_location, QString image_suffix)
```

Comment:
Please pass the `QString` by `const` reference please for consistencies sake :slightly_smiling_face: 

BTW, once we drop the Hyperkit code completely, then this function can go away since none of the backend drivers need to boot the kernel directly.